kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UIServicesClient.java

```diff
@@ -123,6 +123,41 @@
      */
     String getTaskFormByType(String containerId, Long taskId, String formType);
 
+    /**
+     * Returns task form for given task id that belongs to given container as specified user. It returns default form type
+     * which is (FORM - build with form modeler). If there is a need to select the type use #getProcessFormForUserByType
+     * Introduced to allow client to take advantage of bypass auth user connecting to server.
+     * @param userId userId making the request to bypass auth user
+     * @param containerId container identifier where task resides
+     * @param taskId unique task id
+     * @param language language that form should be filtered for
+     * @return  string representation (json or xml depending on client marshaling selection) of the task form
+     */
+    String getTaskFormAsUser(String userId, String containerId, Long taskId, String language);
```

Comment:
Can you please move the userId parameter, to make him for example last one (similar as is done for example in UserTaskServicesClient)? Our convention here is to keep containerId as first parameter.

Reply:
Ok, I'll do that change right away.
Could you take a look at test results to see what's breaking them?

Reply:
there is just one test failure, seems to be an unstable test. Lets see what will be the result once you implement the changes and job will retrigger.